src/components/NumberInput/NumberInput-test.js

```diff
@@ -237,14 +249,22 @@ describe('NumberInput', () => {
         expect(wrapper.state().value).toEqual(90);
       });
 
-      it('should invoke onClick when down arrow is clicked', () => {
+      it('should not invoke onClick when down arrow is clicked and value is 0', () => {
```

Comment:
this was a false positive in the tests before because value wasn't 0. Resetting in between tests exposed this test's weakness